### PR TITLE
zh_cn translation

### PR DESCRIPTION
**Pull request by Catchmons**

### CODE REVIEW

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,91 @@
+{
+
+	"_": "->------------------------]  Game Elements  [------------------------<-",
+
+	"block.railways.semaphore": "壁板式信号机",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"block.railways.semaphore": "壁板信号机",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+	"item.railways.yellow_conductor_cap": "黄色列车长帽",
+	"item.railways.yellow_incomplete_conductor_cap": "黄色列车长帽（半成品）",
+
+	"itemGroup.railways": "机械动力：汽鸣铁道",
+
+	"railways.ponder.conductor_constructing.header": "创建一个指挥机器人",
+	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成。",
+	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
+	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮。",
+	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拨动拉杆。",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆。",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件。",
+	"railways.ponder.conductor_toolboxing.header": "使用指挥机器人运输工具箱",
+	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能正常",
+	"railways.ponder.train_semaphore.header": "使用壁板式信号机可视化信号状态",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.header": "使用壁板信号机可视化信号状态",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.train_semaphore.text_1": "将壁板式信号机放置在列车信号机上方的杆上",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_1": "将壁板信号机放置在列车信号机上方的杆上",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.train_semaphore.text_2": "壁板式信号机下方的杆可由不同材料组成",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_2": "壁板信号机下方的支柱可由不同材料组成",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.train_semaphore.text_3": "壁板式信号机会实时显示杆下信号机的信号状态",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_3": "壁板信号机会实时显示支柱下信号机的信号状态",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 7]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.train_semaphore.text_4": "如果黄铜信号区间有多个有效的出口路径...",
+	"railways.ponder.train_semaphore.text_5": "...可以将第二个壁板式信号机放置在第一个的上方",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_5": "...可以将第二个壁板信号机放置在第一个的上方",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.train_semaphore.text_6": "如果有任一可用信号区间被阻塞，底部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_7": "如果所有信号区间都被阻塞，顶部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_8": "当两个壁板式信号机都放置在非黄铜信号机上时，它们会同时关闭",
```
Comment (iamliuzy) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_8": "当两个壁式信号机都放置在非黄铜信号机上时，它们会同时关闭",
```

**Review by iamliuzy — CHANGES_REQUESTED** [review]
还不错，建议使用专业术语“臂板信号机”

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 9]
```diff
@@ -0,0 +1,91 @@
+{
+
+	"_": "->------------------------]  Game Elements  [------------------------<-",
+
+	"block.railways.semaphore": "壁板信号机",
```
Comment (ChuijkYahus) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"block.railways.semaphore": "臂板信号机",
```

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能正常",
+	"railways.ponder.train_semaphore.header": "使用壁板信号机可视化信号状态",
+	"railways.ponder.train_semaphore.text_1": "将壁板信号机放置在列车信号机上方的杆上",
+	"railways.ponder.train_semaphore.text_2": "壁板信号机下方的支柱可由不同材料组成",
+	"railways.ponder.train_semaphore.text_3": "壁板信号机会实时显示支柱下信号机的信号状态",
+	"railways.ponder.train_semaphore.text_4": "如果黄铜信号区间有多个有效的出口路径...",
+	"railways.ponder.train_semaphore.text_5": "...可以将第二个壁板信号机放置在第一个的上方",
+	"railways.ponder.train_semaphore.text_6": "如果有任一可用信号区间被阻塞，底部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_7": "如果所有信号区间都被阻塞，顶部的信号机将会关闭",
+	"railways.ponder.train_semaphore.text_8": "当两个壁式信号机都放置在非黄铜信号机上时，它们会同时关闭",
```
Comment (ChuijkYahus) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.header": "使用臂板信号机可视化信号状态",
	"railways.ponder.train_semaphore.text_1": "将臂板信号机放置在列车信号机上方的杆上",
	"railways.ponder.train_semaphore.text_2": "臂板信号机下方的支柱可由不同材料组成",
	"railways.ponder.train_semaphore.text_3": "臂板信号机会实时显示支柱下信号机的信号状态",
	"railways.ponder.train_semaphore.text_4": "如果黄铜模式信号区间有多个有效的出口路径……",
	"railways.ponder.train_semaphore.text_5": "……可以将第二个臂板信号机放置在第一个的上方",
	"railways.ponder.train_semaphore.text_6": "如果有任一可用区间被阻塞，底部的臂板信号机将会关闭",
	"railways.ponder.train_semaphore.text_7": "如果所有区间都被阻塞，顶部的臂板信号机将会关闭",
	"railways.ponder.train_semaphore.text_8": "当两个臂板信号机都放置在非黄铜信号机上时，它们会同时关闭",
```
wrong character & proper ellipsis

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+	"item.railways.yellow_incomplete_conductor_cap": "黄色列车长帽（半成品）",
+
+	"itemGroup.railways": "机械动力：汽鸣铁道",
+
+	"railways.ponder.conductor_constructing.header": "创建一个指挥机器人",
+	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成。",
+	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
+	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮。",
+	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆。",
+	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件。",
```
Comment (ChuijkYahus) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.conductor_constructing.text_1": "指挥机器人由安山机壳组成",
	"railways.ponder.conductor_redstoning.header": "借助指挥机器人远程操控红石元件",
	"railways.ponder.conductor_redstoning.text_1": "指挥机器人可以在玩家注视它时按下按钮",
	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆",
	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件",
```
The original translation for Create has no periods in ponder lines.

File: src/generated/resources/assets/railways/lang/zh_cn.json [hunk 12]
```diff
@@ -0,0 +1,91 @@
...
+	"railways.ponder.conductor_redstoning.text_2": "它同样可以在玩家注视它时拉动拉杆",
+	"railways.ponder.conductor_redstoning.text_3": "当玩家戴着与其颜色不一致的列车长帽时，指挥机器人不会激活红石元件",
+	"railways.ponder.conductor_toolboxing.header": "使用指挥机器人运输工具箱",
+	"railways.ponder.conductor_toolboxing.text_1": "工具箱装备在指挥机器人身上时，其功能正常",
+	"railways.ponder.train_semaphore.header": "使用臂板信号机可视化信号状态",
+	"railways.ponder.train_semaphore.text_1": "将臂板信号机放置在列车信号机上方的杆上",
+	"railways.ponder.train_semaphore.text_2": "臂板信号机下方的支柱可由不同材料组成",
+	"railways.ponder.train_semaphore.text_3": "臂板信号机会实时显示支柱下信号机的信号状态",
+	"railways.ponder.train_semaphore.text_4": "如果黄铜信号区间有多个有效的出口路径...",
+	"railways.ponder.train_semaphore.text_5": "...可以将第二个臂板信号机放置在第一个的上方",
```
Comment (ChuijkYahus) [inline src/generated/resources/assets/railways/lang/zh_cn.json]:
```suggestion
	"railways.ponder.train_semaphore.text_4": "如果黄铜信号区间有多个有效的出口路径……",
	"railways.ponder.train_semaphore.text_5": "……可以将第二个臂板信号机放置在第一个的上方",
```

**Comment by techno-sam** [comment]
Please move this file to `src/main/resources/assets/railways/lang/`, if you put it in src/generated, it will be deleted the next time we run datagen

**Comment by Catchmons** [comment]
I think it's done right now

**Review by techno-sam — APPROVED** [review]
Looks good, thanks for translating!
If it isn't an inconvenience to you, please check back once in a while to see if there are missing translations.

**Pull request merged**